modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -52,9 +52,9 @@ a|The power consumption mode.
 
 Possible values:
 
-* `default`
-* `low-latency`
-* `ultra-low-latency`
+* `default`: CPU partitioning with enabled power management and basic low-latency.
```

Comment:
I wonder if we need to mention that default means setting no value or is this a given.